commands/create_user.ts

```diff
@@ -0,0 +1,27 @@
+import { args, BaseCommand } from '@adonisjs/core/ace'
+import User from '#models/user'
+
+export default class CreateUser extends BaseCommand {
+  static commandName = 'create:user'
+  static description = 'Create a new user in the system'
+
```

Comment:
@czaja307 
Try adding startApp param here. It boots the application and enables more functionalities. 
```suggestion
static options: CommandOptions = {
    startApp: true
  }
```

More info here https://docs.adonisjs.com/guides/ace/creating-commands#command-metadata